Docs: added docs for trimLeft, trimRight, trimBoth (#7924)
ClickHouse/ClickHouse: docs/en/query_language/functions/string_functions.md

└───────────────────────────────────┘
```

## trimLeft {#trimleft}

> Removes whitespace from the beginning of a string. Doesn't removes non-standard whitespace characters (tab, no-break space, etc.).

Reviewer: ```suggestion
Removes all consecutive occurrences of common whitespace (ASCII character 32) from the beginning of a string. It doesn't remove other kinds of whitespace characters (tab, no-break space, etc.).
```

> Removes whitespace from the beginning of a string. Whitespace in this context is all the whitespace characters (space, tab, no-break space, etc.).

Reviewer: Doesn't seem to be true:
```
SELECT trimLeft('\t1')
FORMAT CSV

"	1"
```

**Syntax** 

```sql
trimLeft()
```

Alias: `ltrim`.

**Parameters** 

- `string` — string to trim. [String](https://clickhouse.yandex/docs/en/data_types/string/).

**Returned value**

> A string without any leading whitespaces.

Reviewer: ```suggestion
A string without leading common whitespaces.
```

> A string without any heading whitespaces.

Reviewer: ```suggestion
A string without any leading whitespaces.
```

Type: `String`.

**Example**

Query:

```sql

> SELECT concat(trimLeft('     Hello,     '), ' world!')

Reviewer: why do we need `concat` in this example? It'd be simpler with just string literal
